charts/app/templates/formsWebApp.yaml

```diff
@@ -87,6 +87,8 @@ spec:
                 secretKeyRef:
                   key: value
                   name: akv-fwa-session-key
+            - name: SUBDOMAIN_OFFSET
+              value: {{ .Values.formsWebApp.config.server.subdomainOffset | quote }}
```

Comment:
Can you change this to `.Values.formsWebApp.config.subdomainOffset`.

This will also need to be added to the `/charts/app/values.yaml` file. Please add `subdomainOffset` to `formsWebApp.config` with the value `""` (assuming you want the blank value in there. Should look like this:

```
formsWebApp:
  replicaCount: 1
  service:
    name: forms-web-app
    port: 9000
  image:
    repository: pinscommonukscontainers3887default.azurecr.io/forms-web-app
    pullPolicy: Always
    tag: latest
  config:
    port: 3000
    enableLimitedRouting: false
    googleAnalyticsId: ""
    logLevel: trace
    subdomainOffset: ""
    upload:
      debug: true
      uploadDir: /upload-dir
      useTmpFiles: true
```

Reply:
Thanks, actioned:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/721/files#diff-77fd39d9308b9be0e0d91a0a4d7a6be3a197ee124593fb9b25e2e1eef4074dffR91

and

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/721/files#diff-dfbca39ca13a7dd62bff1bd65a1ea1aa719d16eb5ccda48ef12a2eead0bf865fR87